engine/table/src/main/java/io/deephaven/engine/table/impl/AddOnlyToBlinkTableAdapter.java

```diff
@@ -42,14 +45,17 @@ public static Table toBlink(@NotNull final Table table) {
             return table;
         }
 
+        Table addOnlyTable;
```

Comment:
I think it might be safer/less error-prone to make `table` non-final, rather than introduce a new name for the table to operate on.

Reply:
The Java compiler complained about local variable referenced in lambda must be final/effective final

Reply:
I have verified that `table` is never used after `addOnlyTable` is assigned.